std_msgs/Header header
GlobalDetection[] detections
